src/integrators/sdirk_integrators.jl

```diff
@@ -445,7 +430,7 @@ end
 
   cache.ηold = η
   cache.newton_iters = iter
-  @. u = uprev + 2*z
+  @. u += z + dz
```

Comment:
I don't think this is right? `u` is aggregated differently here than in the stages. It's tough to understand if this is correct.